src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -2029,6 +2030,46 @@ public void validateDeleteRequest(
     return rowCounts;
   }
 
+  private static final String exportToParquetTemplate =
+      "export data OPTIONS( "
+          + "uri='<exportPath><table>-*.parquet', "
+          + "format='PARQUET') "
+          + "AS select * from `<project>.<snapshot>.<table>`";
```

Comment:
Do we need to worry about reserved names?  Like, do we need to export from each column explicitly (and quote it)?